### DISCUSSION TITLE
Let PLC know snap7 is connected

### DISCUSSION THREAD

**Discussion by Chup123** [gijzelaerr/python-snap7#398] [Q&A] [answered]
Is it possible to let TIA portal know if snap7 is connected or not, just like other IO?

**Comment by PinataLewis** [comment, marked as answer]
you can simply just write a boolean tag in some DB in PLC using snap7 to indicate it, I assume...

**Comment by swamper123** [comment]
I can remember, that the number of Clients of the protocol snap7 is using is limited on a plc. I haven't got TIA at the moment, but perhaps there is a functionality?

**Comment by PinataLewis** [comment]
您好，邮件已收到。谢谢！

**Comment by gijzelaerr** [comment]
Closing as answered. The recommended approach is to write a boolean heartbeat tag to a DB in the PLC to indicate connectivity.
